internal/state/graph.go

```diff
@@ -233,7 +236,9 @@ func bindHTTPRouteToListeners(
 
 			l, exists := listeners[name]
 			if !exists {
-				r.InvalidSectionNameRefs[name] = struct{}{}
+				// FIXME(pleshakov): Add a proper condition once it is available in the Gateway API.
```

Comment:
Is this tracked as an issue? Let's start creating FIXME and issues atomically going forward.

Reply:
added issues and added links to them in the code